src/swagger/paths.json

```diff
@@ -0,0 +1,116 @@
+{
+    "paths": {
+        "/documents": {
+            "post": {
+                "summary": "Store a credential.",
+                "description": "Store a credential in the storage service.",
+                "requestBody": {
+                    "content": {
+                        "application/json": {
+                            "schema": {
+                                "$ref": "./components/schemas.json#/components/schemas/Credential"
+                            },
+                            "examples": {
+                                "example1": {
+                                    "$ref": "./components/examples.json#/components/examples/ExampleRequest1"
+                                },
+                                "example2": {
+                                    "$ref": "./components/examples.json#/components/examples/ExampleRequest2"
+                                }
+                            }
+                        }
+                    }
+                },
+                "responses": {
+                    "201": {
+                        "description": "Credential stored successfully.",
+                        "content": {
+                            "application/json": {
+                                "schema": {
+                                    "type": "object",
+                                    "properties": {
+                                        "uri": {
+                                            "type": "string",
+                                            "example": "http://localhost:3333/v1/verifiable-credentials/123e4567-e89b-12d3-a456-426614174000.json"
+                                        },
```

Comment:
I noticed that the `API_VERSION` in the config is hardcoded as 'v1.0.0'. I think the uri example here should be updated to reflect this accordingly.